js/modules/modules.go

```diff
@@ -15,8 +17,9 @@ const extPrefix string = "k6/x/"
 
 //nolint:gochecknoglobals
 var (
-	modules = make(map[string]interface{})
-	mx      sync.RWMutex
+	modules        = make(map[string]interface{})
+	moduleVersions = make(map[string]string)
```

Comment:
Would it make sense to reuse the `modules` map, and have its values be a small struct like:

```go
type module struct {
	mod     interface{}
	version string
}
```

This way `getPackageVersion()` would only return the version, as its name implies, and not also set (another) global map value.

This will complicate things slightly, as you'll need to change wherever `GetJSModules()` is called (or change `GetJSModules()` itself to continue to return just the modules without the version; though I think the first approach is slightly better), but it should avoid the repetition, and be a bit cleaner in the end. It would also avoid the need for a separate `GetJSModuleVersions()` function.

Reply:
looks good